.github/build/main.py

```diff
@@ -0,0 +1,77 @@
+import yaml
+import os
+import requests
+
+def main():
+    with open("./config/opendatahub-io/org.yaml", "r") as f:
+        data = yaml.load(f, Loader=yaml.FullLoader)
+        new_odh_org = data.get("orgs").get("opendatahub-io")
+    os.system("git fetch --all")
+    os.system("git checkout main")
+
+    with open("./config/opendatahub-io/org.yaml", "r") as f:
+        data = yaml.load(f, Loader=yaml.FullLoader)
+        old_odh_org = data.get("orgs").get("opendatahub-io")
+
+    pr_number = os.environ.get("PR_NUMBER")
+    affected_teams = get_affected_teams(old_odh_org, new_odh_org)
+    approver_teams = get_valid_approvers(old_odh_org, affected_teams)
+    pr_approvers = get_pr_approvers(pr_number, os.environ.get("TOKEN"))
+    for team in approver_teams:
+        if team.intersection(pr_approvers):
+            print(f"Valid reviewer found for {team} in PR approvers: {pr_approvers}")
```

Comment:
You would have to change the return format of `get_valid_approvers` a good bit to achieve this so I won't block on your doing this, but I _think_ that `team` here would be a list of usernames rather than a team name and that's a bit annoying. It would be nicer if the print statement used the team name.